invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html

```diff
@@ -29,7 +25,8 @@
   <div class="ui relaxed grid">
     <div class="two column row top-padded">
       <div class="eleven wide column">
-        {%- block record_body %}
+        {%- block record_body -%}
+        {%- block record_header -%}
```

Comment:
Blocks are added so that the export page and others can be made much simpler (see export.html)